server/pages/index.jade

```diff
@@ -118,13 +118,15 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 		if i18nLocaleScript
 			script(src=i18nLocaleScript)
 		if 'development' === env || isDebug
+			script(src=urls[ 'polyfill' ])
```

Comment:
does this work? I would suspect it has to come _after_ manifest? Though I guess no one requires it, so ...